tests/util/dune

```diff
@@ -1,5 +1,6 @@
 (library
   (name testutil)
   (wrapped false)
-  (libraries core oUnit fileutils scilla-base)
+  (libraries core batteries oUnit fileutils scilla-base patdiff.lib scilla-util)
```

Comment:
I am getting build errors due to `patdiff.lib` in this line.

Reply:
Have you installed `patdiff` library? ```opam install patdiff```